### ISSUE TITLE
Slider Invert, could this only change the percentages and not the displayed position?

### ISSUE THREAD

**Issue by krasatos** [custom-cards/slider-button-card#20]
It's not a bug, request or a problem, Ill try to explain the situation:

My cover entity for my rollers is showing 100% when it's closed.
Meaning when it is fully unrolled, the position is 100.
This picture is at 30%
![image](https://user-images.githubusercontent.com/1773624/169951227-9fa91d39-896e-4283-8d3f-c9e933da3af5.png)
![image](https://user-images.githubusercontent.com/1773624/169951292-a4d14a4f-748c-405a-a840-ccfc6ac10239.png)

At the same time, the slider-button-card shows the inverted percentage, but the correct position:
![image](https://user-images.githubusercontent.com/1773624/169951395-a96295d0-92fe-4109-b6fc-d7d201c22cfd.png)

If i enable `slider: inverted: false`, I get the correct percentage but the wrong depiction
![image](https://user-images.githubusercontent.com/1773624/169951370-84d79a7f-0fa3-4e01-aee7-ee12c948d652.png)

Is there some way to invert only the percentage displayed on the card, while keeping the graphic the same?

Perhaps two different invert options, one for the percentage and one for the slider?
Or is there something else i'm missing?

Thank you in advance!

PS. I reposted this from the previous github page, hope it's ok.

**Comment by danielbrunt57** [comment]
Your issue is the same as mine but with a vertical slat blind.  I guess in your case you've tried switching between top-bottom and bottom-top?  I suspect the issue then becomes the visual representation of the blind is wrong...

**Comment by danielbrunt57** [comment]
I agree there needs to be two invert options, one for the visual and another for the percentage.

**Comment by bbadaboom** [comment]
I second that! Would be great to have an option for an invert in the shown percentage separatly!

**Comment by Barnakill** [comment]
I know this is an old topic, but I'm "reviving" it to also second that it would be a great update to be able to configure it separately because many times it's confusing to have 2 different percentages for the same device.
Thank you very much in advance for your time and selfless dedication.

Greetings and sorry for my English.

**Comment by danielbrunt57** [comment]
> I know this is an old topic, but I'm "reviving" it to also second that it would be a great update to be able to configure it separately because many times it's confusing to have 2 different percentages for the same device. Thank you very much in advance for your time and selfless dedication.

I concur!!
